CORRELATION_CONTEXT_HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,48 @@
+# Correlation Context HTTP Header Format
+
+A correlation context header is used to pass the name-value context properties for the trace. This is a companion header for the `Trace-Context`. The values should be passed along to any child requests. Note that uniqueness of the key within the `Correlation-Context` is not guaranteed. Context received from upstream service may be altered before passing it along.
+
+# Format
+
+## Header name
+
+`Correlation-Context`
+
+Multiple correlation context headers are allowed. Values can be combined in a single header according to the [rfc](https://tools.ietf.org/html/rfc7230#page-24).
+
+## Header value
+
+`name1=value1[;properties1],name2=value2[;properties2]`
+
+**Limits:**
+1. Maximum number of name-value pairs: `180`.
+2. Maximum number of bytes per a single name-value pair: `4096`.
+3. Maximum total length of all name-value pairs: `8192`.
+
+*See [CORRELATION_CONTEXT_HTTP_HEADER_FORMAT_RATIONALE.md#Limits] for rationale of these limits.*
+
+## Name format
+
+Url encoded string. Spaces should be trimmed from beginning and the end of the name. Names are case sensitive.
```

Comment:
I have a concern about the `case sensitive` as some frameworks out there `normalize` headers making them lowercase. What about require a lowercase so we avoid these situations?

Reply:
I don't think that is the case @jcchavezs. Headers name may be normalized but I am pretty sure the values cannot be normalized as lowercase because that will completely change the value in case of base64 encoded strings for example.

Reply:
Maybe I am missing something but aren't this going to be non-encoded names?

Reply:
both names and values are on the right-hand side of the header, I agree with @bogdandrutu

Reply:
I see, then forget about my doubt.